src/kani-compiler/rustc_codegen_kani/src/overrides/hooks.rs

```diff
@@ -178,7 +178,7 @@ impl<'tcx> GotocHook<'tcx> for Assert {
             // [KANI_REACHABILITY_CHECK] <check ID>
             // for reachability checks
             msg = format!("[{}] {}", assert_id, msg);
-            let reach_msg = format!("[KANI_REACHABILITY_CHECK] {}", assert_id);
+            let reach_msg = GotocCtx::reach_check_msg(&assert_id);
```

Comment:
Long term do we want to keep this in the hooks? Or is this solved by writing our own assertion macro that we inject?

Reply:
I'm thinking we should keep this in the hooks for the long term. Our own assertion macro calls this hook here: https://github.com/model-checking/kani/blob/541a3100d88d42bbad941060490c723f87313c53/library/std/src/lib.rs#L38